README.md

```diff
@@ -40,6 +40,10 @@ This will also generate the files `header.tab` and `header.txt`,
 which can be used for loading the output into other programs
 for further processing.
 
+### Install
```

Comment:
This commit was part of another pull request.  Please remove it from this pull request (e.g. with `rebase -i` and then `push -f`).

Reply:
Done